dbt/seeds/location/docs.md

```diff
@@ -0,0 +1,29 @@
+# neighborhood_group
+
+{% docs seed_neighborhood_group %}
+A mapping of neighborhood codes to groups that are used for sales
+validation.
+
+These mappings are defined by the Valuations team and then delivered to the
+Data team in the form of an Excel workbook. The data team then manually
```

Comment:
```suggestion
Data team in the form of an Excel workbook. The Data team then manually
```

Reply:
Good catch, fixed in 8c4fcde6bf6c7b5cb7ca769b21983047852b9f56.